modules/ROOT/pages/runtime-urls-allowlist.adoc

```diff
@@ -1,9 +1,10 @@
-= Ports, IPs, and Hostnames Whitelist
+= Ports, IPs, and Hostnames to Allow
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:page-aliases: runtime-urls-whitelist.adoc
 
-In your network, you may need to allowlist the hostnames and ports of various parts of Anypoint Platform to allow API Gateway capabilities of the runtime to communicate with the MuleSoft-managed online Anypoint Platform APIs and services.
+You may need to add the Anypoint Platform hostnames and ports to an allowlist in your network so that the API Gateway capabilities of the runtime can communicate with the online Anypoint Platform APIs and services that are managed by MuleSoft.
```

Comment:
may > might, globally (unless actually giving permission)
```suggestion
You might need to add your network Anypoint Platform hostnames and ports to an allowlist so that the API Gateway capabilities of the runtime can communicate with the online Anypoint Platform APIs and services that are managed by MuleSoft.
```
"the runtime" = Mule or some other runtime?

Reply:
In context it is clear